lib/analytics.js

```diff
@@ -92,6 +94,21 @@ Analytics.prototype.addIntegration = function(Integration) {
   return this;
 };
 
+/**
+ * Define a new `IntegrationMiddleware`
+ *
+ * @param {Function} Middleware
+ * @return {Analytics}
+ */
+
+Analytics.prototype.addIntegrationMiddleware = function(Middleware) {
```

Comment:
lowercase for the argument `middleware` is probably fine? (Integration is capital because it's more of a constructor that contstructs integration instances)